Trello_items.py

```diff
@@ -0,0 +1,78 @@
+from secrets import KEY, TOKEN
+import requests
+
+todo_boardid = "5f6076c34c5f48265943e31e"
+todolistid = "5f6076e68cc021208da06d2b"
+donelistid = "5f6076e96994a166c05385a6"
+
+apiurl = "https://api.trello.com/1/"
+boardsurl = apiurl + 'boards/'
+getcardsonboardsurl = boardsurl + todo_boardid + '/cards'
+cardsurl = apiurl + 'cards/'
+
+def getcardsonlist_URL(cardid):
```

Comment:
This function name is misleading. It's actually for finding the list of a given card but sounds like it's getting the cards on a given list.